src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -113,6 +111,11 @@
 
         <Grid.ContextMenu>
             <ContextMenu Style="{StaticResource WorkspaceContextMenuStyle}">
+                <ContextMenu.Resources>
+                    <Style TargetType="{x:Type MenuItem}">
+                        <Setter Property="Focusable" Value="False" />
+                    </Style>
+                </ContextMenu.Resources>
```

Comment:
when hover on menu item inside context menu, focus is removed from search textbox. With this change menu items cannot receive focus anymore and focus stays in in-canvas search textbox